### PR TITLE
added URL to my repo

### PR DESCRIPTION
**Pull request by muddybeard210**

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -5,24 +5,24 @@
 //
```
Comment (sjschmidt44) [inline script.js]: Looks like the commented out code should be available for use in the game function?

File: script.js [hunk 2]
```diff
@@ -5,24 +5,24 @@
...
-//
-// for (var i = 0; i < questions.length; i += 1) {
-//   game(questions[i], answers[i]);
-// }
-//
-// function sayHi() {
-//   var hiTest = prompt("This is a test.  Say Hi");
-//   document.write(hiTest);
+function game(x, y) {
+  var ques = prompt(questions[i]).toLowerCase();
```
Comment (sjschmidt44) [inline script.js]: You're passing in `x, y` but not using them anywhere in the function.

File: script.js [hunk 3]
```diff
@@ -5,24 +5,24 @@
...
+}
+
+for (var i = 0; i < questions.length; i += 1) {
+  game(questions[i], answers[i]);
+}
+
+function sayHi() {
+  var hiTest = prompt("This is a test.  Say Hi");
+  document.write(hiTest);
 // }
```
Comment (sjschmidt44) [inline script.js]: I also don't see where you've called the game function anywhere?

Comment (lillianszugyi) [inline script.js]: @sjschmidt44 sorry - this is not my final script file, so I think I tried, then deleted the game function, or I never called it. My final document calls a couple different functions. :-\

**Pull request merged**